dnf5/commands/list/list.cpp

```diff
@@ -58,12 +57,8 @@ void ListCommand::set_argument_parser() {
     specs->set_complete_hook_func([&ctx](const char * arg) { return match_specs(ctx, arg, true, true, false, false); });
     cmd.register_positional_arg(specs);
 
-    auto showduplicates = parser.add_new_named_arg("showduplicates");
-    showduplicates->set_long_name("showduplicates");
-    showduplicates->set_description("Show all versions of the packages, not only the latest ones.");
-    showduplicates->set_const_value("true");
-    showduplicates->link_value(&config.get_showdupesfromrepos_option());
-    cmd.register_named_arg(showduplicates);
+    show_duplicates = std::make_unique<libdnf::cli::session::BoolOption>(
+        *this, "showduplicates", '\n', "Show all versions of the packages, not only the latest ones.", false);
```

Comment:
I am just wondering, why is the short name for the command == '\n'. Does it have some special hidden meaning?

Reply:
To be honest, I've copy/pasted it from somewhere. But good question, I'll investigate.

Reply:
Well, it should be `\0`

Reply:
So it looks like it wasn't copy/paste, but my invention. Thanks for spotting this!. Fixed.